angular-todo/src/app/views/login/login.component.ts

```diff
@@ -17,10 +23,43 @@ import { Component } from '@angular/core';
   selector: 'app-login',
   templateUrl: './login.component.html',
 })
-export class LoginComponent {
+export class LoginComponent implements OnInit{
   isWebAuthn = false;
+  code: string;
+  state: string;
+
+  get isCentrlizedLogin(): boolean {
```

Comment:
can we spell `isCentralizedLogin` correctly? Is there a reason for this spelling?

Reply:
no reason, my bad, it got away from me.

Reply:
the above comment was meant for the callback comment i made earlier so my bad i posted it here